plugins/idea/pluginBase/src/main/java/com/imaginea/kodebeagle/base/object/WindowObjects.java

```diff
@@ -38,7 +37,7 @@
     private Editor windowEditor;
     private int distance;
     private int size;
-    private String esURL;
+    private String kbAPIURL;
     private Map<String, String> fileNameContentsMap = new HashMap<String, String>();
     private Map<String, List<Integer>> fileNameNumbersMap = new HashMap<String, List<Integer>>();
     private Map<String, String> repoStarsMap = new HashMap<String, String>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2293)